src/components/smartSearch/filters/SubQuery/index.tsx

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Why is this required? Disabling linting rules entirely takes away the benefit of having them in the first place. There can of course be exceptions, but they should be specific for each occasion and not cover entire modules.

I realize taking a closer look at existing files that this has been done before and I haven't caught it. So maybe this is another something to put into the refactor issue?

Reply:
Added to #337